app/src/features/rules/components/RulesList/components/RulesTable/hooks/useRuleTableColumns.tsx

```diff
@@ -149,7 +193,7 @@ const useRuleTableColumns = (options?: Record<string, boolean>) => {
               </Row>
             ),
           },
-        ].filter((option) => (!isRule ? option.key !== 1 : option.key !== 0));
+        ].filter((option) => (!isRule ? option.key !== 1 : true));
```

Comment:
Can we add a `disabled` with value `!isRule` property instead of using `key`, and filter using this property.